### PR TITLE
REF: validate-table uses click

### PR DESCRIPTION
**Pull request by ebolyen**
This is the best I could manage without rewriting a lot of code.

Basically just a crude wrapper around what used to be a `Command` class.

### CODE REVIEW

File: biom/cli/table_validator.py [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+import numpy as np
+
+from biom.util import HAVE_H5PY, biom_open, is_hdf5_file
+
+def validate_table(input_fp, format_version=None, detailed_report=False):
+    result = TableValidator()(table=input_fp, format_version=format_version,
+                              detailed_report=detailed_report)
+    return result['valid_table'], result['report_lines']
+
+# The below is almost unfixable in it's current form.
```
Comment (wasade) [inline biom/cli/table_validator.py]: just add an issue please

Comment (ebolyen) [inline biom/cli/table_validator.py]: Will do

**Comment by wasade** [comment]
just one comment, :+1: otherwise

**Comment by ebolyen** [comment]
@wasade added issue #664

**Comment by ebolyen** [comment]
Fixed merge conflict and updated comment to point at issue.

**Comment by wasade** [comment]
Thanks!

**Pull request merged**